docs/WindowsBuild.md

```diff
@@ -131,6 +131,8 @@ cmake -B "S:\b\toolchain" ^
   -D SWIFT_WINDOWS_x86_64_ICU_I18N="S:/Library/icu-64/usr/lib/icuin64.lib" ^
   -D CMAKE_INSTALL_PREFIX="C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr" ^
   -D PYTHON_EXECUTABLE=C:\Python27\python.exe ^
+  -D PYTHON_LIBRARIES="C:\Program Files\Python37\libs\python37.lib" ^
+  -D PYTHON_INCLUDE_DIRS="C:\Program Files\Python37\include" ^
```

Comment:
Should we refer to the the Python3 that is installed via Visual Studio (as explained above)?

Reply:
Oh indeed -- updated the instructions.

Reply:
So, you changed my `python37.lib` to `python3.lib`. That does not work for me:

```
cmd.exe /C "cd . && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe" -E vs_link_dll --intdir=tools\lldb\source\API\CMakeFiles\liblldb.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100177~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100177~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1426~1.288\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\liblldb.rsp  /out:bin\liblldb.dll /implib:lib\liblldb.lib /pdb:bin\liblldb.pdb /dll /version:10.0 /INCREMENTAL:NO /DEBUG /OPT:REF /OPT:ICF /INCREMENTAL:NO   && cd ."
LINK: command "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1426~1.288\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\liblldb.rsp /out:bin\liblldb.dll /implib:lib\liblldb.lib /pdb:bin\liblldb.pdb /dll /version:10.0 /INCREMENTAL:NO /DEBUG /OPT:REF /OPT:ICF /INCREMENTAL:NO /MANIFEST /MANIFESTFILE:bin\liblldb.dll.manifest" failed (exit code 1104) with the following output:
LINK : fatal error LNK1104: cannot open file 'python37.lib'
```

Reply:
I suspect that the `python3.lib` was just an accident and was supposed to be `python37.lib`.

The `python3` interpreter cannot be used yet because there are dependencies on python2 but LLDB requires Python3 though.  Thats why there is the explicit Python2 and Python3 usage.  `PYTHON_EXECUTABLE` should be removed (there is an existing change up to do that but it needs more changes in LLVM/clang that need to be pulled down) IMO as it makes it very confusing and disallows Python2/Python3 usage simultaneously.